test-cases/tests/products/h21-verify-all-products-using-the-workloadwebapp.md

```diff
@@ -0,0 +1,47 @@
+---
+estimate: 30m
+---
+
+# H21 - Verify all products using the workload-web-app
```

Comment:
is this a manual test case or automated test case? It should be automated as it is relatively easy to add it to our pipelines. @STEPHINRACHEL has done for one already.

Reply:
Is manual, but if we have if it's already in the pipeline than we can remove it, does the pipeline also verify that the products are working or only deploy the workload-web-app?